packages/api/src/magic.link.js

```diff
@@ -0,0 +1,25 @@
+export const createMagicTestTokenBypass = (
```

Comment:
Just curious: what's the motivation for exporting the "create bypass" function? Will we ever need to make one elsewhere?

Reply:
@yusefnapora Mostly just abstraction. Also doing it as a function might make it easier to unit test if things get complex.

Reply:
ah, I missed the bit where you can override the default bypass in the AuthorizationTestContext constructor. Seems like a good idea, since it sounds like there's going to be an "official" magic link bypass & we might have other auth providers with their own bypass details